<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BOOK-4QTRLAP57K" failures="0" tests="4" name="sevenMartPackage.ManageNewsTest" time="9.856" errors="2" timestamp="2025-05-07T07:41:50 IST" skipped="2">
  <testcase classname="sevenMartPackage.ManageNewsTest" name="addNewsInManageNewsModule" time="2.023">
    <skipped/>
  </testcase> <!-- addNewsInManageNewsModule -->
  <system-out/>
  <testcase classname="sevenMartPackage.ManageNewsTest" name="searchAnyNews" time="2.259">
    <error message="C:\Users\shanu\eclipse-workspace\7rMartSupermarketProject\src\test\resources\TestData.xlsx (The system cannot find the path specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\shanu\eclipse-workspace\7rMartSupermarketProject\src\test\resources\TestData.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:159)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:17)
at sevenMartPackage.ManageNewsTest.searchAnyNews(ManageNewsTest.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </error>
  </testcase> <!-- searchAnyNews -->
  <system-out/>
  <testcase classname="sevenMartPackage.ManageNewsTest" name="addNewsInManageNewsModule" time="1.919">
    <error message="C:\Users\shanu\eclipse-workspace\7rMartSupermarketProject\src\test\resources\TestData.xlsx (The system cannot find the path specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\shanu\eclipse-workspace\7rMartSupermarketProject\src\test\resources\TestData.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:159)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:17)
at sevenMartPackage.ManageNewsTest.addNewsInManageNewsModule(ManageNewsTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </error>
  </testcase> <!-- addNewsInManageNewsModule -->
  <system-out/>
  <testcase classname="sevenMartPackage.ManageNewsTest" name="addNewsInManageNewsModule" time="3.655">
    <skipped/>
  </testcase> <!-- addNewsInManageNewsModule -->
  <system-out/>
</testsuite> <!-- sevenMartPackage.ManageNewsTest -->
